code/src/main/java/com/onlinecontacttracing/storage/data-storage/Constants.java

```diff
@@ -7,6 +7,7 @@
   final static long SIX_HOURS_IN_SECONDS = 21600L;
   final static long ONE_WEEK_IN_SECONDS = 604800L;
   final static long TWO_WEEKS_IN_SECONDS = 1209600L;
+  final static long TWO_WEEKS_IN_MILLI_SECONDS = 1209600000L;
```

Comment:
nit: here and above, it's nicer to see this written as a multiplication rather than a long. EX:
```
final static long TWO_WEEKS_IN_MILLI_SECONDS = 2 * 7 * 24 * 60 * 60 * 1000L;
```

Reply:
done